components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -1018,143 +1018,146 @@ export default class PluginManagement extends AdminSettings<Props, State> {
                     >
                         {this.renderEnablePluginsSetting()}
 
-                        <BooleanSetting
-                            id='requirePluginSignature'
-                            label={
-                                <FormattedMessage
-                                    id='admin.plugins.settings.requirePluginSignature'
-                                    defaultMessage='Require Plugin Signature:'
-                                />
-                            }
-                            helpText={
-                                <FormattedMarkdownMessage
-                                    id='admin.plugins.settings.requirePluginSignatureDesc'
-                                    defaultMessage='When true, uploading plugins is disabled and may only be installed through the Marketplace. Plugins are always verified during Mattermost server startup and initialization. See [documentation](!https://mattermost.com/pl/default-plugin-signing) to learn more.'
-                                />
-                            }
-                            value={this.state.requirePluginSignature}
-                            disabled={this.props.isDisabled || !this.state.enable}
-                            onChange={this.handleChange}
-                            setByEnv={this.isSetByEnv('PluginSettings.RequirePluginSignature')}
-                        />
-                        <BooleanSetting
-                            id='automaticPrepackagedPlugins'
-                            label={
-                                <FormattedMessage
-                                    id='admin.plugins.settings.automaticPrepackagedPlugins'
-                                    defaultMessage='Enable Automatic Prepackaged Plugins:'
-                                />
-                            }
-                            helpText={
-                                <FormattedMarkdownMessage
-                                    id='admin.plugins.settings.automaticPrepackagedPluginsDesc'
-                                    defaultMessage='When true, automatically installs any prepackaged plugin found to be enabled in the server configuration.'
+                        { !this.props.config.ExperimentalSettings.RestrictSystemAdmin && (
```

Comment:
This was the only line added to this file. Github is not great handling all the indentation differences.